platforms/documentation/docs/src/docs/userguide/dep-man/advanced/variant_aware_resolution.adoc

```diff
@@ -366,6 +367,75 @@ Gradle knows the _consumer_ requires a Java 7-compatible library, but the _produ
 If the _consumer_ requested `org.gradle.jvm.version=15`, Gradle could choose either the Java 8 or Java 11 *variant*.
 Gradle would then select the highest compatible version—Java 11.
 
+[[sec:secondary-variants]]
+== Secondary Variants
+
+**Secondary variants** are *additional sets of artifacts* that a variant can expose, which do **not** participate in graph resolution.
+Instead, they are considered *only during artifact selection*.
+
+A secondary variants:
+
+* Is an additional artifact set exposed by a configuration.
+* It has its own attributes, artifacts, and optionally capabilities.
+* It must be explicitly published (secondary variants are not usually published or declared in metadata).
```

Comment:
This is not true. Artifact sets of a configuration are _published automatically_ unless [explicitly skipped](https://github.com/gradle/gradle/blob/d83bd4b2c135e14d0dfe865df9e620779386ae31/subprojects/core-api/src/main/java/org/gradle/api/component/ConfigurationVariantDetails.java#L39)

Reply:
Updated to: * Is published automatically unless explicitly skipped.